tests/test.ts

```diff
@@ -142,6 +143,37 @@ describe.skip('Test clasp deploy function', () => {
   });
 });
 
+describe.skip('Test clasp versions function', () => {
+  it('should list versions correctly', () => {
+    const result = spawnSync(
+      'clasp', ['versions'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('~ ');
+    expect(result.stdout).to.contain(' Versions ~');
+    expect(result.status).to.equal(0);
+  });
+});
+
+describe.skip('Test getScriptURL function from utils', () => {
+  it('should return the scriptURL correctly', () => {
+    const url = getScriptURL('abcdefghijklmnopqrstuvwxyz');
+    expect(url).to.equal('https://script.google.com/d/abcdefghijklmnopqrstuvwxyz/edit');
+    const url2 = getScriptURL('1234567890abcdefg');
+    expect(url2).to.equal('https://script.google.com/d/1234567890abcdefg/edit');
+  });
+});
+
+describe.skip('Test getFileType function from utils', () => {
+  it('should return the lowercase file type correctly', () => {
+    const type = getFileType('SERVER_JS');
```

Comment:
This function only appears in `index.ts` once. We could probably inline it, as all it does is a `toLowerCase` unless the filetype is `SERVER_JS`, where it then just converts it to `js`

Reply:
This function is fine standalone. It's good to test it for coverage.